tests/cli/test_install_tar.sh

```diff
@@ -0,0 +1,142 @@
+lang en_US.UTF-8
+timezone America/New_York
+keyboard us
+rootpw redhat
+bootloader --location=mbr
+zerombr
+clearpart --initlabel --all
+autopart
+
+liveimg --url http://10.0.2.2/root.tar.xz
+__EOF__
+
+        tmp_dir=(mktemp -d /tmp/composer.XXXXX)
+        ssh_key_dir=$(mktemp -d /tmp/composer-ssh-keys.XXXXXX)
+
+        rlRun -t -c "ssh-keygen -t rsa -N '' -f $ssh_key_dir/id_rsa"
+        pub_key=(cat $ssh_key_dir/id_rsa.pub)
+
+        bp_name="test-tar"
+        blueprint="$bp_name.toml"
+        cat > $blueprint << __EOF__
+name = "$bp_name"
+description = "tar image test"
+version = "0.0.1"
+modules = []
+
+# temporary
+[[packages]]
+name = "grubby-deprecated"
+version = "*"
+
+[[packages]]
+name = "kernel"
+version = "*"
+
+[[packages]]
+name = "openssh-server"
+version = "7.*"
+
+[[groups]]
+name = "anaconda-tools"
+
+[[customizations.user]]
+name = "root"
+key = "$pub_key"
+__EOF__
+        composer-cli blueprints push $blueprint
+        image_path="/var/www/html/root.tar.xz"
+
+        version=$(awk -F = '$1 == "VERSION_ID" { print $2 }' /etc/os-release)
+        arch=$(uname -m)
+        baseurl=$(curl "https://mirrors.fedoraproject.org/mirrorlist?repo=fedora-${version}&arch=${arch}" | \
+            grep -v "^#" | head -n 1)
+        curl --remote-name-all $baseurl/images/pxeboot/{vmlinuz,initrd.img}
+
+        fallocate -l 5G disk.img
+    rlPhaseEnd
+
+    rlPhaseStartTest "compose start"
+        rlAssertEquals "SELinux operates in enforcing mode" "$(getenforce)" "Enforcing"
+        uuid=$($CLI compose start $bp_name tar)
+        rlAssertEquals "exit code should be zero" $? 0
+
+        uuid=$(echo $uuid | cut -f 2 -d' ')
+    rlPhaseEnd
+
+    rlPhaseStartTest "compose finished"
+        if [ -n "$uuid" ]; then
+            until $CLI compose info $uuid | grep 'FINISHED\|FAILED'; do
+                sleep 10
+                rlLogInfo "Waiting for compose to finish ..."
+            done;
+        else
+            rlFail "Compose uuid is empty!"
+        fi
+
+        rlRun -t -c "$CLI compose image $uuid"
+        image="$uuid-root.tar.xz"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Install tar image using kickstart liveimg command"
+        cat >> $ks_path << __EOF__
+%post
+echo -e "curl http://10.0.2.2/image-$uuid-booted\n" >> /etc/rc.d/rc.local
+chmod +x /etc/rc.d/rc.local
+%end
+__EOF__
+        mv $image $image_path
+        restorecon $image_path
+        $QEMU_BIN -m 2048 -drive file=disk.img,format=raw -nographic -monitor none -net user,id=nic0,hostfwd=tcp::2222-:22 \
+            -net nic -kernel vmlinuz -initrd initrd.img \
+            -append "inst.ks=http://10.0.2.2/ks-tar.cfg inst.stage2=$baseurl console=ttyS0" &
+        timeout=$((60*60)) # wait up to 60 minutes
+        t=0
+        while ((t < timeout)) && ! grep -q "image-$uuid-booted" /var/log/httpd/access_log; do
+            rlLogInfo "Waiting for the tar image to be installed and system to boot up."
+            sleep 10
```

Comment:
IMO you can increase sleeps to 60 everywhere b/c these operations take a long time, though not strictly necessary.